### PR TITLE
Aref translation api

### LINKED ISSUE #15: Test available APIs for text translation
Test and determine the best text translation solution

### PR DESCRIPTION
**Pull request by Aref-Sadeghi-Gh**
Translation class is ready

### CODE REVIEW

File: CogNative/backend/translation/translation.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from googletrans import Translator
+
+
+class translation:
+
+    def __int__(self):
```
Comment (andrew-fennell) [inline CogNative/backend/translation/translation.py]: __int__()  ->  __init__()

without this correction, the class won't instantiate self.translator...

File: CogNative/backend/translation/translation.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from googletrans import Translator
```
Comment (andrew-fennell) [inline CogNative/backend/translation/translation.py:1]: Add the correct version of googletrans in requirements.txt

Please add a _version_! When I installed googletrans with pip the default version didn't run this code properly.

File: CogNative/backend/translation/translation.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+class translation:
+
+    def __int__(self):
+        self.translator = Translator()
+
+    def translate(self, text, src, des):
+        result = self.translator.translate(text, src=src, dest=des)
+        return result.text
+
+    def translate_to(self, text, des):
```
Comment (andrew-fennell) [inline CogNative/backend/translation/translation.py:15]: You might add a docstring to explain how this functions different. I'm assuming t just auto-detects src language?

**Review by andrew-fennell — CHANGES_REQUESTED** [review]
A few changes need to be made.

**Comment by Aref-Sadeghi-Gh** [comment]
fixed

**Review by andrew-fennell — APPROVED** [review]
LGTM

**Pull request merged**